android/res/values/strings.xml

```diff
@@ -1128,6 +1128,8 @@
 	<string name="power_managment_setting_never">Never</string>
 	<string name="power_managment_setting_auto">Automatic</string>
 	<string name="power_managment_setting_manual_max">Maximum power saving</string>
+	<string name="enable_screen_sleep">Allow screen to sleep</string>
```

Comment:
Please update data/strings.txt and run `./tools/unix/generate_locationzation.sh` instead.

Reply:
Assuming you mean data/strings/strings.txt and that you're OK with Google Translate generated values? Re: UX please review https://github.com/organicmaps/organicmaps/issues/196 which I hope gives a good explanation. It's critical that complete control over battery use is available on longer bike or hiking trips where you don't have regular (daily) access to electricity. I've tested this a few times and the application is impractical to use if you have to keep manually switching the screen off. See https://the-bikepacker.blogspot.com/search/label/Trips for some examples :-)

Reply:
p.s. I use a Bluetooth paired watch as a trusted device and this stops the screen from actually locking. That way after the display sleeps, if I need to see the map again, I just tap the screen to wake it up. Easy to do when cycling.

Reply:
@rtsisyk Hi, any tips for setting up Ruby to run the localization script? I installed using brew and moved the new version to the front of my PATH (I'm on a Mac). When I run the script I get 

```
<internal:/usr/local/Cellar/ruby/3.0.1/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in `require': cannot load such file -- safe_yaml/load (LoadError)
	from <internal:/usr/local/Cellar/ruby/3.0.1/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /Users/martin/AndroidStudioProjects/organicmaps/tools/twine/lib/twine/plugin.rb:2:in `<top (required)>'
	from <internal:/usr/local/Cellar/ruby/3.0.1/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from <internal:/usr/local/Cellar/ruby/3.0.1/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /Users/martin/AndroidStudioProjects/organicmaps/tools/twine/lib/twine.rb:25:in `<module:Twine>'
	from /Users/martin/AndroidStudioProjects/organicmaps/tools/twine/lib/twine.rb:1:in `<top (required)>'
	from <internal:/usr/local/Cellar/ruby/3.0.1/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from <internal:/usr/local/Cellar/ruby/3.0.1/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from ./tools/unix/../../tools/twine/bin/twine:2:in `<main>'
```
and
```
martin@Raticate organicmaps % which ruby
/usr/local/opt/ruby/bin/ruby
martin@Raticate organicmaps % ruby --version
ruby 3.0.1p64 (2021-04-05 revision 0fb782ee38) [x86_64-darwin20]
```
Thanks

Reply:
There are some workarounds in google, I don't remember which one helped me.

Reply:
OK, I'll hit Google. Never worked with Ruby before.